src/analysis/problematic_exclusions.py

```diff
@@ -105,7 +104,7 @@ def populate_labels(
     terms = [x for x in terms if any([x.startswith(y) for y in prefix_map.keys()])]
 
     # Instantiate template
-    prefix_sparql_strings = [f'prefix {k}: <{v}>' for k, v in prefix_map.items()]
+    prefix_sparql_strings = [f'prefix {k}: <{v}>\n' for k, v in prefix_map.items()]
```

Comment:
why isnt this solved on the level of the template? This is a list of prefix strings, right? The template engine should be able to turn a list of strings into a newline separated string output?

Reply:
We can. I think it will look something like `{% for string in prefix_sparql_strings %}{{string}}{-% endfor %}` which makes the thing harder to read, so in this particular case I like this better.

Also for time purposes I'd like to leave as is, but after looking at my explanation, if you you do still want this logic moved into the Jinja template, let me know and I can do.

Reply:
Ok, its fine; I am just noting that, from a software engineering perspective, to ensure seperation of concerns, the layer that is responsible for defining how the final string (the whole sparql query) should be represented is the template, while the prefix_maps iteration is still on the data side. In any case, technically the whole list comprehension could be in the template, but from my POV this is just academic. I am satisfied with the explanation given that the hole list comprehension is conceptually part of the templating system.